cmd/commento/main.go

```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	_ "github.com/mattn/go-sqlite3"
+	"github.com/joho/godotenv"
 	
 	. "github.com/adtac/commento/lib"
```

Comment:
I actually like those dot imports when they are used sparingly but, I recognize they are a discouraged practice. Almost all (probably all) dot imports will be removed eventually.
I include the dot imports to avoid changing too much of the code at once.

Without the dot import there would be, for example, references to 
lib.LoadDatabase instead of LoadDatabase. But if the package changes then all the libs have to change to that new package name which is annoying.
When the structure of the code is more settled the dots will get removed. Really, they are just there now for ease of readability (that's why I like them even if I know they have to go...)